inc/Engine/Optimization/Minify/CSS/Minify.php

```diff
@@ -323,9 +347,30 @@ protected function minify( $file_path, $minified_file, $file_content ) {
 	 * @since 3.1
 	 *
 	 * @param string $file_content Content to minify.
+	 *
 	 * @return Minifier\CSS
 	 */
 	protected function get_minifier( $file_content ) {
 		return new Minifier\CSS( $file_content );
 	}
+
+	/**
+	 * Get external files excluded from minify.
+	 *
+	 * @since 3.7.6
+	 *
+	 * @return array Excluded external files.
+	 */
+	private function get_excluded_externals() {
+		/**
+		 * Filters CSS external URLs to exclude from the minify process
+		 *
+		 * @since 3.7.6
+		 *
+		 * @param array $pattern Patterns to match.
+		 */
+		$excluded_externals = (array) apply_filters( 'rocket_minify_css_excluded_external', [] );
+
+		return array_merge( $excluded_externals, $this->options->get( 'exclude_css', [] ) );
```

Comment:
I don't like using array_merge as it's slow, can u check the following and test it.
```suggestion
		return [ ...$excluded_externals, ...$this->options->get( 'exclude_css', [] ) ];
```
This code is using [Spread Operator](https://wiki.php.net/rfc/spread_operator_for_array) which I think it's included from PHP 5.6 (If I remember well)

and also do we need to make this array unique?
if Yes, I think using `array_flip` twice will do the trick faster.

@iCaspar What do u think?

Reply:
Found a way to eliminate this method altogether :)